lib/acts_as_taggable_on/compatibility.rb

```diff
@@ -1,11 +1,7 @@
 module ActsAsTaggableOn::Compatibility
```

Comment:
@seuros it's used [here](https://github.com/mbleigh/acts-as-taggable-on/blob/master/lib/acts_as_taggable_on/tagger.rb#L18-L23), so if we want to drop this module, we have to change API or extract parser for options